kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/container/signature/ValidationResult.java

```diff
@@ -0,0 +1,73 @@
+package org.eclipse.kura.container.signature;
```

Comment:
Copyright header missing

Reply:
DAMMIT! One day I will open a PR with all the copyright headers first try I promise

Reply:
Addressed in c5c668e11cc023457cfb0ed269a2ce508504fd51